Gems/ScriptCanvas/Code/Editor/View/Windows/MainWindow.cpp

```diff
@@ -1256,7 +1256,7 @@ namespace ScriptCanvasEditor
 
     void MainWindow::RemoveScriptCanvasAsset(const SourceHandle& assetId)
     {
-        AssetHelpers::PrintInfo("RemoveScriptCanvasAsset : %s", assetId.ToString().c_str());
+        AZ_TracePrintf("ScriptCanvas", "RemoveScriptCanvasAsset : %s", assetId.ToString().c_str());
```

Comment:
This should add a newline.
AZ_TracePrintf doesn't automatically add a newline unlike AZ_Assert, AZ_Error or AZ_Warning